plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/AbstractTestHiveMetastore.java

```diff
@@ -38,7 +38,7 @@
 @Execution(SAME_THREAD)
 abstract class AbstractTestHiveMetastore
 {
-    private HiveMetastore metastore;
+    protected HiveMetastore metastore;
```

Comment:
Introducing `getMetastore()` seems more appropriate.
Your changes only access for reading the metastore.